tmt/steps/provision/testcloud.py

```diff
@@ -301,6 +301,9 @@ def go(self) -> None:
                 self.info('disk', f"{value} GB", 'green')
             elif key == 'connection':
                 self.verbose('connection', value, 'green')
+            elif key == 'key':
+                if value:
+                    self.verbose('key', '\n'.join(value), 'green')
```

Comment:
I'm not sure that this line can be hit with current possibilities however the message should be 'info' instead (before the patch it is printed by 'value is not None' which prints 'info')

Reply:
Used `self.info` instead.